config/samples/manifests.yaml

```diff
@@ -17,6 +17,10 @@ spec:
       containers:
       - name: runner
         image: actions-runner:latest
+        ports:
+          - name: metrics
+            containerPort: 8080
+            protocol: TCP
```

Comment:
Do we always need to specify the metrics port in a custom resource?
Is it possible for the controller to automatically specify the port?

Reply:
That's good.

I plan to rewrite the reconcile loop significantly, in https://github.com/cybozu-go/github-actions-controller/pull/25.
I will adopt your suggestion there.